packages/coordinator/src/screens/bookedDonation/BookedAppointmentScreen.tsx

```diff
@@ -57,3 +71,15 @@ export default function BookedAppointmentScreen(
     </CoordinatorScreen>
   );
 }
+
+function nameBar(name: string) {
```

Comment:
Change to `NameBar(props: {name: string}){ ... }` and call it like `<NameBar name="XXX"/>`.

You can also set NameBar to accept `BookedAppointmentScreenProps` and than call it `<NameBar {...props}/>` (because looks like you need multiple parent props

Reply:
Plus, when icons are clicked you need to call the relevant functions (see in the props of this component)